assets/src/settings-page/analytics.js

```diff
@@ -125,18 +129,15 @@ function AnalyticsEntry( { entryIndex, onChange, onDelete, type = '', config = '
 				}
 			</h4>
 			<div className="amp-analytics-entry__options" id={ `amp-analytics-entry-${ String( entryIndex ) }` }>
-				<div className="amp-analytics-entry__text-inputs">
-					<TextControl
-						className="option-input"
+				<div className="amp-analytics-entry__select-control">
+					<SelectControl
+						className="option-select"
 						label={ __( 'Type:', 'amp' ) }
 						onChange={ ( newType ) => {
 							onChange( { type: newType } );
 						} }
-						type="text"
-						pattern="^[a-zA-Z0-9_.-]+$"
-						title={ __( 'Must be a valid vendor or left blank for in-house analytics.', 'amp' ) }
-						placeholder={ __( 'Vendor or blank', 'amp' ) }
 						value={ type }
+						options={ AnalyticsVendorsOptions }
```

Comment:
The `AnalyticsVendorsOptions` variable name makes it look like a class/component. Should it rather be `analyticsVendorsOptions`?

Reply:
Yes it should be as you suggested, will surely make the cahnges.

Reply:
resolved